### PR TITLE
Add ERC: Lockable Extension for ERC-1155

### PR DESCRIPTION
**Pull request by piyush-chittara**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: ERCS/eip-locakable1155.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+---
+eip:
```
Comment (abcoathup) [inline ERCS/eip-locakable1155.md:2]:
```suggestion
eip: 7721
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

File: ERCS/eip-locakable1155.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+title: Lockable Extension for ERC-1155
+description: Interface for enabling locking of ERC-1155 using locker and token id based approvals
+author: Piyush Chittara (@piyush-chittara)
+discussions-to: https://ethereum-magicians.org/t/lockable-extension-for-erc1155/20250
```
Comment (abcoathup) [inline ERCS/eip-locakable1155.md:6]:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-7721-lockable-extension-for-erc1155/20250
```

File: ERCS/eip-locakable1155.md (hunk 1 above)
Comment (piyush-chittara) [reply ERCS/eip-locakable1155.md]: thanks for eip number, updated!

**Comment by abcoathup** [comment]
@piyush-chittara need to rename the filename to `erc-7721.md`

**Comment by piyush-chittara** [comment]
thanks @abcoathup, it's done!

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**